project.clj

```diff
@@ -41,5 +42,11 @@
                                   [org.clojure/tools.nrepl "0.2.13" :exclusions [org.clojure/clojure]]]
                    :source-paths ["dev"]
                    :repl-options {:init-ns user}}
+             :test {:dependencies [[io.pedestal/pedestal.service-tools "0.5.3"]
+                                  [com.stuartsierra/component.repl "0.2.0"]
+                                  [org.clojure/tools.namespace "0.3.0-alpha4"]
+                                  [org.clojure/tools.nrepl "0.2.13" :exclusions [org.clojure/clojure]]]
+                    :source-paths ["dev"]
```

Comment:
Only the ` :source-paths ["dev"]` is necessary (for now), delete the other ones.

Learn about profiles here: https://github.com/technomancy/leiningen/blob/master/doc/PROFILES.md